kratos/python_scripts/analysis_stage.py

```diff
@@ -346,7 +346,7 @@ def __CreateListOfProcesses(self):
 
     def __CheckIfSolveSolutionStepReturnsAValue(self, is_converged):
         """In case the solver does not return the state of convergence
-        (same as the SolvingStrategy does) then issue ONCE a deprecation-warning
+        (same as the ImplicitSolvingStrategy does) then issue ONCE a deprecation-warning

```

Comment:
```suggestion
        (same as the SolvingStrategy does) then issue ONCE a deprecation-warning

```

this was correct as is

Reply:
Agree. Thanks for spotting!